lib/matplotlib/axis.py

```diff
@@ -316,6 +316,17 @@ def set_label2(self, s):
         self.label2.set_text(s)
         self.stale = True
 
+    def set_url(self, s):
+        """
+        Set the url of label1 and label2
+
+        ACCEPTS: str
```

Comment:
shouldn't this use normal numpydoc?

Reply:
Indeed, from an old PR. I've also updated the argument name to match the name from the overridden method.